File: Python Word examples/paragraphs/updateParagraphText.py
import spirecloudword
from spirecloudword import ParagraphFormat, Color, Border, Borders
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.paragraphs_api.ParagraphsApi(configuration)

file_name = "Template.docx"
index = 1
node_path = "sections/0"
text = "Test Update Paragraph Text "
extra = {'folder': 'input', 'dest_file_path': 'output/UpdateParagraphText.docx'}
result = api.update_paragraph_text(file_name, node_path, index, text, **extra)

File: JavaScript Word Examples/ParagraphsApiDemo/updateParagraphText.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.ParagraphsApi(apiClient);

    var filename = "UpdateParagraphText.docx";
    var nodePath = "Section/0/Body/0";
    var opt = {
        folder: 'input',
    };
    var index = 0;
    var text = "Updated Paragraph 2 ";
    var deFilePath = "output/UpdateParagraphText_ouput.docx";
    instance.updateParagraphText(filename, nodePath, index, text, deFilePath, opt, function (error, data, response) {
        if (error) throw error;    
    });
})();